build.gradle

```diff
@@ -34,13 +34,14 @@ buildscript {
 }
 
 ext.versions = [
-        flink  : "1.14.2",
-        hadoop : "2.7.7",
-        junit4 : "4.11",
-        junit5 : "5.4.+",
-        mockito: "2.0.+",
-        slf4j  : "1.7.0",
-        vavr   : "0.9.2",
+        flink   : "1.14.2",
+        hadoop  : "2.7.7",
+        junit4  : "4.11",
+        junit5  : "5.4.+",
+        mockito : "2.0.+",
+        mockito3: "3.+",
+        slf4j   : "1.7.0",
+        vavr    : "0.9.2",
```

Comment:
nit: maybe we should try to not align the colons given that this is constantly going to change based on the longest module name.

Reply:
I think editor config might help streamline these. I will take a look.